docs/introduction.rst

```diff
@@ -1,73 +1,51 @@
 .. include:: defs.rst
 .. _introduction:
 
+********
 Overview
-========
-|AppName| is open source scientific image processing software integrating hardware control, data acquisition, visualization, processing, and analysis using Python. |AppName| is easily extended using Python. It runs on Windows, Linux, and macOS.
+********
+Nion Swift is open source scientific image processing software for controlling instruments, and managing the acquisition, organization, visualization, and analysis of data. The software is designed to be used in conjunction with electron microscopes. However, it is also useful in other data visualization applications. Nion Swift helps you visualize your data both while it is being acquired and during processing and analysis.
 
-|AppName| is intended to provide an extensible platform to acquire scientific data from a variety of microscopes and other instruments, be intuitive and easy to use, and allow sophisticated data visualization, processing, and analysis.
+Click to download Nion Swift or for :ref:`installation` instructions.
 
-Key Features
+Click here to get started fast with our :ref:`basic-use`.
 
-- Data handling of 1D plots, 2D images, 1D and 2D collections of plots and images, and sequences.
-- Live computations that can be chained during acquisition or live parameter adjustment.
-- Open source, cross platform (macOS, Windows, Linux), and Python based scientific data processing.
-- Low latency, high throughput data acquisition platform.
+User Interface
+==============
+The user interface is comprised of two main sections: a central section called the workspace, and the surrounding area for utility panels.
 
-Data Items and Display Items
-----------------------------
-|AppName| operates on data arrays, which include 2D images and 1D spectra, which are generally called *Data Items*.
+.. image:: graphics/workspace.png
+  :width: 320
+  :alt: Example of the Swift Workspace 
 
-Data items represent data arrays with anywhere from one to five dimensions. The data array associated with a data item can be 1D or 2D data and then organized into 1D or 2D collections or 1D sequences.
+The central workspace is for display panels which can either display a single data item or thumbnail images of multiple data items in one of the browser formats. For more information, see :ref:`display-panels`.
```

Comment:
This could be reworded to use active voice: "The central workspace shows display panels..." or something like that.